parser/parser.y

```diff
@@ -3335,9 +3336,33 @@ DefaultValueExpr:
 	NowSymOptionFraction
 |	SignedLiteral
 |	NextValueForSequence
+|	BuiltinFunction
+
+BuiltinFunction:
+	'(' BuiltinFunction ')'
+	{
+		$$ = $2.(*ast.FuncCallExpr)
+	}
+|	identifier '(' ')'
+	{
+		$$ = &ast.FuncCallExpr{
+			FnName: model.NewCIStr($1),
+		}
+	}
+|	identifier '(' ExpressionList ')'
+	{
+		$$ = &ast.FuncCallExpr{
+			FnName: model.NewCIStr($1),
+			Args:   $3.([]ast.ExprNode),
+		}
+	}
 
 NowSymOptionFraction:
-	NowSym
+	'(' NowSymOptionFraction ')'
+	{
+		$$ = $2.(*ast.FuncCallExpr)
+	}
+|	NowSym
```

Comment:
This change is not compatible with MySQL:

```sql
create table t (a int, b timestamp default now() on update (now()));
```

Maybe we can revert this and remove `NowSymOptionFraction` from `DefaultValueExpr`, and convert the `"NOW"` string to `"CURRENT_TIMESTAMP"` in `setFuncCallDefaultValue()`.